### PR TITLE
Paid implementation

### PR DESCRIPTION
**Pull request by TheLillyleaf**

### CODE REVIEW

**Review by TheLillyleaf — COMMENTED** [review]
self check

File: Payment/Expense/index.spec.ts [hunk 1]
```diff
@@ -4,10 +4,12 @@ describe("Payment.Expense", () => {
 	const expense: issuefab.Payment.Expense = {
 		type: "expense",
 		limit: [10, "EUR"],
-		paid: {},
+		paid: { abcd0001: { issuer: "tobe@kakal.cse", created: "2021-12-20T13:37:42Z", modified: "2021-12-20T13:37:42Z" } },
```
Comment (linusarnejohansson) [inline Payment/Expense/index.spec.ts:7]: do we want to create an additional test where the timestamps are not the same?

**Review by EliasEriksson — APPROVED** [review]

File: Payment/index.spec.ts [hunk 2]
```diff
@@ -5,14 +5,16 @@ describe("Payment", () => {
 		type: "card",
 		limit: [10, "EUR"],
 	}
-	const expense: issuefab.Payment = {
+	const expense: issuefab.Payment.Expense = {
 		type: "expense",
 		limit: [10, "EUR"],
+		paid: { abcd0001: { issuer: "tobe@kakal.cse", created: "2021-12-20T13:37:42Z", modified: "2021-12-20T13:37:42Z" } },
```
Comment (linusarnejohansson) [inline Payment/index.spec.ts:11]: same question here, do want a test with different timestamps?

File: Payment/Expense/index.spec.ts (hunk 1 above)
Comment (TheLillyleaf) [reply Payment/Expense/index.spec.ts:7]: We could, but we don't check that anywhere. Today, we don't use Created at all, and modified is only tested in the API.

**Review by linusarnejohansson — APPROVED** [review]
sehr gut

**Pull request merged**